internal_packages/keybase/lib/key-manager.cjsx

```diff
@@ -0,0 +1,35 @@
+{Utils, React} = require 'nylas-exports'
+PGPKeyStore = require './pgp-key-store'
+KeybaseUser = require './keybase-user'
+kb = require './keybase'
+_ = require 'underscore'
+
+module.exports =
+class KeyManager extends React.Component
+  @displayName: 'KeyManager'
+
+  @propTypes:
+    keys: React.PropTypes.array.isRequired
+
+  constructor: (props) ->
+    super(props)
+
+  render: ->
+    {keys} = @props
+
+    keys = _.map(keys, (key) =>
+      if key.key?
+        uid = "key-manager-" + key.key.get_pgp_fingerprint().toString('hex')
+      else if key.keybase_user?
+        uid = "key-manager-" + key.keybase_user.components.username.val
+      else
+        uid = "key-manager-" + key.addresses.join('')
+      return <KeybaseUser profile={key} key={uid} />
+    )
+
+    if not keys? or keys.length < 1
```

Comment:
I don't think there should ever be a scenario where `not keys?` since _.map will always return an array!